app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/ui/theme/Theme.kt

```diff
@@ -0,0 +1,38 @@
+package au.com.gridstone.trainingkotlin.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val DarkColorPalette = darkColors(
+  primary = Primary,
+  primaryVariant = PrimaryDark,
+  secondary = Secondary
+)
+
+private val LightColorPalette = lightColors(
+  primary = Primary,
+  primaryVariant = PrimaryDark,
+  secondary = Secondary
+)
+
+@Composable
+fun TrainingKotlinTheme(
+  darkTheme: Boolean = isSystemInDarkTheme(),
+  content: @Composable () -> Unit,
+) {
+  val colors = if (darkTheme) {
+    DarkColorPalette
+  } else {
+    LightColorPalette
+  }
```

Comment:
no need for braces in a single line expression, according to [style guide](https://github.com/Gridstone/KotlinStyleGuide#if-statements)
```kotlin
  val colors = 
    if (darkTheme) DarkColorPalette
    else LightColorPalette
```